src/backend/marsha/core/api/video.py

```diff
@@ -153,12 +153,15 @@ def get_permissions(self):
                     & (permissions.IsTokenInstructor | permissions.IsTokenAdmin)
                 )
             ]
-        elif self.action in ["destroy"]:
+        elif self.action in ["destroy", "bulk_destroy"]:
             # Not available in LTI
             # For standalone site, only playlist admin or organization admin can access
             permission_classes = [
-                permissions.IsObjectPlaylistAdminOrInstructor
-                | permissions.IsObjectPlaylistOrganizationAdmin
+                permissions.IsAuthenticated
```

Comment:
Why is it needed to add `IsAuthenticated` permission ?

Reply:
Because I wanted this test : https://github.com/openfun/marsha/blob/master/src/backend/marsha/core/tests/api/video/test_bulk_destroy.py#LL23C1-L24C1
to keep failing

Reply:
ok but I don't understand why here it was failing and not on the view viewset. I will try to understand later.